### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by stiefel40k** [flask-admin/flask-admin#1632]
Hiho!

According to #1603 you were about to issue a new release which would make flask-admin compatible with sqlalchemy 1.2+. May I ask when is it going to happen? It would be nice, if I wouldn't had to patch flask-admin myself :)

Thanks in advance!

EDIT: I apologize for opening a new issue, but I wasn't sure if I should comment on an already closed issue, which I can't reopen, or open a new one...

**Comment by gordol** [comment]
Can't you just run from master? It's been merged already: https://github.com/flask-admin/flask-admin/pull/1586

**Comment by mrjoes** [comment]
The issue was closed because I pushed the new release 1.5.0 that has a fix.

**Comment by stiefel40k** [comment]
```bash
$ tail flask-admin-1.5.0/flask_admin/contrib/sqla/fields.py 

            field.populate_obj(model, None)

            self.inline_view._on_model_change(field, model, is_created)


def get_pk_from_identity(obj):
    # TODO: Remove me
    cls, key = identity_key(instance=obj)
    return u':'.join(text_type(x) for x in key)
```

but the 1.5.0 doesn't include the fix

**Comment by gordol** [comment]
Yep...

It got blown out it looks like... even though that commit is in the tree, someone else blew it out later.

**Comment by gordol** [comment]
https://github.com/flask-admin/flask-admin/commit/197896a178ca4c569cfd499ab23248730955eddc#diff-9898624cd2b7292472f2a75dd4c4db1c

"fix for python 2 compatibility" broke it again.

**Comment by stiefel40k** [comment]
And yes I can run it from master, but than why use package managers, releases and dependency management at all ;)

It kinda makes the Dokerbuild funky :)

**Comment by gordol** [comment]
Running from master wouldn't actually help in this case, someone blew out the change.

**Comment by gordol** [comment]
Actually the same author of both PRs did it. :) @asakaplan paging @asakaplan <3

**Comment by stiefel40k** [comment]
actually it works fine with py3 as well in the current master version (I tried it), cos the key is still the second in the list, so it's fine, the problem is it is not in the release

**Comment by stiefel40k** [comment]
the only thing is you have no access to the cls and the **token (whatever they are, but they don't seem to be used anyways)

**Comment by cjmayo** [comment]
https://pypi.python.org/pypi/Flask-Admin/1.5.1#downloads

**Comment by SBillion** [comment]
I don't understand how you can run it from master. It stills broken:

https://github.com/flask-admin/flask-admin/blob/master/flask_admin/contrib/sqla/fields.py#L299

**Comment by asakaplan** [comment]
Hi all, sorry I've been busy with a new job. I submitted the pull request originally with the * syntax and fixed it to work with py2.x after the 2.x tests failed. 

Am I correct in saying that it was just a versioning issue y'all are having, or is this still an issue?

**Comment by stiefel40k** [comment]
I originally opened the issue only because of versioning, and I can tell for sure, that the fix works on py3, and I highly suspect it works on py2 as well. So no it is not an issue.

**Comment by SBillion** [comment]
@stiefel40k The fix is not in master or in 1.5.1
